better-font-awesome.php

```diff
@@ -128,13 +133,13 @@ class Better_Font_Awesome_Plugin {
      *
      * @return  Better_Font_Awesome  The BFA object.
      */
-    public static function get_instance( $args = '' ) {
-        static $instance = null;
-        if ( null === $instance ) {
-            $instance = new static( $args );
-        }
+    public static function get_instance() {
+
+	    if ( ! self::$instance ) {
+		    self::$instance = new self();
```

Comment:
Use ```self::$instance = new static();```` if we don't care about 5.2 support. It allows better abstraction handling. With ```self::$instance = new self();``` you don't have to override the get_instance method after abstracting the class for it to return the correct class instance